docs/deployment/aws-secureboot.md

```diff
@@ -33,20 +36,80 @@ make aws-secureboot-dev BUILD_OPTS="--lessram"
 ```
 
 ## Prepare spawning
-This chapter describes what steps are required to successfully spawn an AWS instance with Garden Linux running with secure boot.
+This chapter describes what steps are required to successfully spawn an AWS instance with Garden Linux running with secure boot. There are two options to choose from.
 
-### UEFI variables
+You can either enable secure boot internally. This way you simply need to spawn the Garden Linux image and execute the `enroll-gardenlinux-secureboot-keys`, which will prepare the Garden Linux image for secure boot by using UEFI's setup mode. After a reboot, the Garden Linux image will then run with secure boot enabled.
```

Comment:
Suggestion:
O: 'After a reboot, the Garden Linux image will then run with secure boot enabled.'
N: 'Ensure to reboot the system to have secure boot fully enabled.'